### PR TITLE
Feat: addedhome page

### PR DESCRIPTION
**Pull request by ianuragsharma**
image

### CODE REVIEW

**Review by ckahub — COMMENTED** [review]
LGTM

File: index.html [hunk 1]
```diff
@@ -0,0 +1,153 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (surbhi84) [inline index.html:7]: author and description meta tags can be included

File: index.html [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+
+<body>
+    <nav class="nav-container flex-row">
+        <a href="../index.html">
+            <h5>ezBuy</h5>
+        </a>
+        <input name="name" type="text" class="input-area text-base " placeholder="Search...">
+        <ul class="flex-row">
+            <li>
+                <a href="">
```
Comment (surbhi84) [inline index.html:21]: why empty link

**Review by surbhi84 — APPROVED** [review]
rest looks good to me.

File: index.html (hunk 1 above)
Comment (ianuragsharma) [reply index.html:7]: ok

File: index.html (hunk 2 above)
Comment (ianuragsharma) [reply index.html:21]: acutally these are login,account,cart and wishlist link, will add them in the end.

**Pull request merged**